test/util/data/bitcoin-util-test.json

```diff
@@ -1,274 +1,226 @@
 [
-  { "exec": "./bitcoin-tx",
+  { "exec": "./bitzeny-tx",
     "args": ["-create", "nversion=1"],
     "output_cmp": "blanktxv1.hex",
     "description": "Creates a blank v1 transaction"
   },
-  { "exec": "./bitcoin-tx",
+  { "exec": "./bitzeny-tx",
     "args": ["-json","-create", "nversion=1"],
     "output_cmp": "blanktxv1.json",
     "description": "Creates a blank v1 transaction (output in json)"
   },
-  { "exec": "./bitcoin-tx",
-    "args": ["-"],
-    "input": "blanktxv2.hex",
-    "output_cmp": "blanktxv2.hex",
-    "description": "Creates a blank transaction when nothing is piped into bitcoin-tx"
```

Comment:
Why do you remove tests?

Reply:
I forgot about it. so I mistaked.
I will add those test fix soon.

Reply:
Because it tests about version2 transaction.